### ISSUE TITLE
[A11y]: Usage of wrong UI elements

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/whiteboard#897]
<img width="1237" height="635" alt="Image" src="https://github.com/user-attachments/assets/4a7fb458-a187-4886-a1e2-b6fb4b147661" />

<img width="900" height="135" alt="Image" src="https://github.com/user-attachments/assets/0ceffacb-23a3-47d8-8d41-30a2ffb7a15b" />

There are multiple problems regarding current implementation: 

- The are multiple form controls with the same id
- Wrong usage of input elements: there are functional buttons and should be used as buttons. Then the problem of navigation flow, focusing and description will be resolved too.
